WebRoot/js/zimbraMail/prefs/ZmPreferencesApp.js

```diff
@@ -77,6 +77,10 @@ ZmPreferencesApp._registerAllPrefs =
 function() {
 	AjxDispatcher.require("PreferencesCore");
 	appCtxt.getAppController().runAppFunction("_registerPrefs");
+	if (appCtxt.get(ZmSetting.ADMIN_DELEGATED) && !appCtxt.get(ZmSetting.ADMIN_MAIL_ENABLED) && appCtxt.get(ZmSetting.ADMIN_PREFERENCES_ENABLED)) {
```

Comment:
when are we calling _registerPrefs when ADMIN_DELEGATED is false?

Reply:
When it is not a delegated admin access, `ZmPreferencesApp._registerPrefs` and `ZmMailApp._registerPrefs` are executed by the following function in the correct order.
```
ZmPreferencesApp._registerAllPrefs =
function() {
	//...
	appCtxt.getAppController().runAppFunction("_registerPrefs");
```